Memories/Memories/VCs/AddViewController.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  AddViewController.swift
+//  Test
+//
+//  Created by Alexander Supe on 13.03.20.
+//  Copyright © 2020 Alexander Supe. All rights reserved.
+//
+
+import UIKit
+import MapKit
+import CoreImage.CIFilterBuiltins
+
+class AddViewController: UIViewController, CLLocationManagerDelegate {
+    
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var titleField: UITextField!
+    @IBOutlet weak var imageButton: UIButton!
+    @IBOutlet weak var filterButton: UIButton!
+    
+    @IBOutlet weak var audioButton: UIButton!
+    @IBOutlet weak var videoButton: UIButton!
+    @IBOutlet weak var audioView: UIView!
+    @IBOutlet weak var videoView: UIView!
+    
+    var titleText: String?
+    var image: Data?
+    var originalImage: Data?
+    var audio: String? { UserDefaults.standard.string(forKey: "CurrentAudio") }
+    var video: String? { UserDefaults.standard.string(forKey: "CurrentVideo") }
+    
+    var picker = UIImagePickerController()
+    let locationManager = CLLocationManager()
+    var coordinates: CLLocationCoordinate2D? { didSet { finishSaving() }}
```

Comment:
I'm not a big fan of squashing all these brackets together, and the spacing is inconsistent. 
Please put brackets on separate lines (the exception is a guard let that return's nothing or nil)

```swift
var coordinate: CLLocationCoordinate2D {
    CLLocationCoordinate2D(latitude: latitude, longitude: longitude)
}
```